doc/convert/interactive.md

```diff
@@ -1,5 +1,8 @@
 # Interactive conversion
 
+***Important notice***: This functionality is currently broken. Interactive browsing
+will be reintegrated in a later update.
+
```

Comment:
let's hope we don't forget to update this note then :)

Reply:
The person who updates the Python code hopefully also documents features here and removes the notice :D